devs/docs/es-backports.rst

```diff
@@ -141,26 +141,26 @@ should be crossed out as well.
 - [ ] 43ace5f80d7 Emit deprecation warnings when boosts are defined in mappings (#62623)
 - [ ] 9f5e95505bf Also abort ongoing file restores when snapshot restore is aborted (#62441) (#62607)
 - [ ] 06d5d360f92 Tidy up fillInStackTrace implementations (#62555)
-- [ ] 9bb7ce0b229 [7.x] Allocate new indices on "hot" or "content" tier depending on data stream inclusion (#62338) (#62557)
-- [ ] 91e23305295 Warn on badly-formed null values for date and IP field mappers (#62487)
+- [s] 9bb7ce0b229 [7.x] Allocate new indices on "hot" or "content" tier depending on data stream inclusion (#62338) (#62557)
+- [s] 91e23305295 Warn on badly-formed null values for date and IP field mappers (#62487)
 - [ ] e0a4a94985f Speed up merging when source is disabled. (#62443) (#62474)
 - [ ] 62dcc5b1ae1 Suppress stack in VersionConflictEngineException (#62433)
 - [ ] 5112c173194 Add WARN Logging on Slow Transport Message Handling (#62444) (#62521)
 - [ ] 14aec44cd86 Log if recovery affected by disconnect (#62437)
-- [ ] 24a24d050ab Implement fields fetch for runtime fields (backport of #61995) (#62416)
+- [s] 24a24d050ab Implement fields fetch for runtime fields (backport of #61995) (#62416)
 - [ ] ffbc64bd109 Log WARN on Response Deserialization Failure (#62368) (#62388)
 - [ ] 95766da3452 Save Some Allocations when Working with ClusterState (#62060) (#62303)
 - [ ] 875af1c976f Remove Dead Variable in BlobStoreIndexShardSnapshots. (#62285) (#62295)
-- [ ] 808c8689ac9 Always include the matching node when resolving point in time  (#61658)
-- [ ] 3fc35aa76e6 Shard Search Scroll failures consistency (#62061)
-- [ ] 4d528e91a12 Ensure validation of the reader context is executed first (#61831)
-- [ ] 3d69b5c41e2 Introduce point in time APIs in x-pack basic (#61062)
+- [s] 808c8689ac9 Always include the matching node when resolving point in time  (#61658)
+- [s] 3fc35aa76e6 Shard Search Scroll failures consistency (#62061)
+- [s] 4d528e91a12 Ensure validation of the reader context is executed first (#61831)
+- [s] 3d69b5c41e2 Introduce point in time APIs in x-pack basic (#61062)
 - [ ] 7b941a18e9d Optimize Snapshot Shard Status Update Handling (#62070) (#62219)
-- [ ] 6710104673d Fix Creating NOOP Tasks on SNAPSHOT Pool (#62152) (#62157)
+- [s] 6710104673d Fix Creating NOOP Tasks on SNAPSHOT Pool (#62152) (#62157)
```

Comment:
Skipped since https://github.com/elastic/elasticsearch/pull/62157/commits/6d328acc6db41cefcb2bd94f16b2ae3a3c21a512 relies on `org.elasticsearch.common.blobstore.DeleteResult` which was introduced in new [repo cleanup api](https://github.com/elastic/elasticsearch/commit/6aaee8aa0ae481611e25e9ff0b7fcd90ca750970)